apps.json

```diff
@@ -1510,6 +1510,16 @@
         ],
         "url": "https://github.com/YunoHost-Apps/mattermost_ynh"
     },
+    "mautrix_whatsapp": {
+        "branch": "master",
+        "category": "communication",
+        "revision": "HEAD",
```

Comment:
Please if you set `HEAD` create and work on the branch `testing`

Reply:
Yep I'll do it just after merging my current PR in master. Then I'll do the next PRs on testing first

Reply:
working version merged and testing branch created.